@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -319,11 +319,7 @@ export function isShaderTechnique(technique: Technique): technique is ShaderTech
  * @param technique Technique to check.
  */
 export function needsVertexNormals(technique: Technique): boolean {
-    return (
-        isStandardTechnique(technique) ||
-        isTerrainTechnique(technique) ||
-        isStandardExtrudedLineTechnique(technique)
-    );
+    return isStandardTechnique(technique) || isStandardExtrudedLineTechnique(technique);
```

Comment:
The code in Tile.ts is only calling this method if there are no normal yet, so no need for this change.
```
                if (!bufferGeometry.getAttribute("normal") && needsVertexNormals(technique)) {
                    bufferGeometry.computeVertexNormals();
                }
```

Reply:
Agreed, https://github.com/heremaps/harp.gl/pull/475